ginkgo.Jenkinsfile

```diff
@@ -34,6 +34,25 @@ pipeline {
                 }
             }
         }
+        stage('Provision'){
+            environment {
+                GOPATH="${WORKSPACE}"
+                TESTDIR="${WORKSPACE}/${PROJ_PATH}/test"
+            }
+            steps {
+                sh 'cd ${TESTDIR}; vagrant up runtime --no-provision'
+                sh 'cd ${TESTDIR}; K8S_VERSION=1.7 vagrant up k8s1-1.7 --no-provision'
+                sh 'cd ${TESTDIR}; K8S_VERSION=1.7 vagrant up k8s2-1.7 --no-provision'
+                sh 'cd ${TESTDIR}; K8S_VERSION=1.6 vagrant up k8s1-1.7 --no-provision'
+                sh 'cd ${TESTDIR}; K8S_VERSION=1.6 vagrant up k8s2-1.7 --no-provision'
+            }
+            post {
+                failure {
+                    sh "cd ${TESTDIR}; vagrant destroy -f"
+                    sh "cd ${TESTDIR}; K8S_VERSION=1.6 vagrant destroy -f"
```

Comment:
needs to have a corresponding `K8S_VERSION=1.7` destroy command.

Reply:
Hi @ianvernon the 1.7 is by default, and it'll be destroyed on `vagrant destroy -f` so it's not needed.

Reply:
Can you still add it? This is not obvious and we may change this default.